### PR TITLE
PRO-2096 - Arka Admin Screen for SQLite

### PR DESCRIPTION
**Pull request by vignesha22**
## Description

- Added SQLite for config changes on cron and refreshed cron time every time its updated
- Removed sponsorAddress from checkWhitelist API
- Added react app to modify SQLite db config variables

## Types of changes
What types of changes does your code introduce?

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Other (please describe): 

## Further comments (optional)

-

### CODE REVIEW

File: admin_frontend/README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
```
Comment (ch4r10t33r) [inline admin_frontend/README.md:1]: Delete this readme and replace it with a proper one.

File: admin_frontend/public/index.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
```
Comment (ch4r10t33r) [inline admin_frontend/public/index.html:1]: Please replace default index.html

File: admin_frontend/public/manifest.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
+{
```
Comment (ch4r10t33r) [inline admin_frontend/public/manifest.json:1]: Replace template names with proper product name. Create React App should be replaced with Arka. Please replace the logo with Arka logo.

**Review by ch4r10t33r — CHANGES_REQUESTED** [review]
Add a common README.md file (which is common to both frontend and backend that demonstrates how to get Arka up and running.

File: README.md [hunk 4]
```diff
@@ -30,6 +30,20 @@ The flow is as follows:
 - [Deposit](https://etherspot.fyi/arka/api-calls/deposit) to the paymaster.
 - Include the paymaster data in the [UserOp](https://etherspot.fyi/account-abstraction/userops) to have the transaction sponsored.
 
+## 🐳 How to run (a Docker image)
+
+## Prerequisites
+* Docker Installed (https://www.docker.com/get-started/)
+* Edit docker-compose.yml for backend cron job and put a value in CRON_PRIVATE_KEY with the wallet private key you want to use to periodically update the paymaster contract price
```
Comment (ch4r10t33r) [inline README.md:37]: Pls have a default value setup. Which will run the docker with default configuration.

**Review by ch4r10t33r — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (vignesha22) [reply README.md:37]: all other env vars are of default values only the private_key is not there but its not mandatory and will skip if there is no private key given inside cron job

**Pull request merged**